app/models/course/user_invitation.rb

```diff
@@ -11,6 +11,7 @@ class Course::UserInvitation < ApplicationRecord
   validate :no_existing_unconfirmed_invitation
 
   enum role: CourseUser.roles
+  enum timeline_algorithm: CourseUser.timeline_algorithms
```

Comment:
Note: For future development when timeline algorithm setting is included in a course user invitation